### DISCUSSION TITLE
difference from shinchiro

### DISCUSSION THREAD

**Discussion by alakittty** [zhongfly/mpv-build#67] [Q&A] [answered]
heya @zhongfly what's the difference between your builds and shinchiro's builds?

**Comment by zhongfly** [comment, marked as answer]
It can be assumed that there is no difference: it's just the difference between compiling at different times with the same compilation settings and using the latest version of the source code at that time.
Only when the compilation fails I will try some small changes to make the compilation successful.

**Reply by zhongfly** [reply]
To be clearer, [shinchiro](https://github.com/shinchiro/mpv-winbuild-cmake) prepares the build system for compiling mpv for windows, and I just use his tools to compile mpv.

**Reply by Infinitytreacher** [reply]
why don't you make a release per new commit instead of for following the same model(1 release per day) as shinchiro (that is redundancy imo). If i want to switch from shinchiro's builds to yours, would the updater.exe work and fetch your latest release?

**Comment by zhongfly** [comment]
> why don't you make a release per new commit instead of for following the same model(1 release per day) as shinchiro (that is redundancy imo). If i want to switch from shinchiro's builds to yours, would the updater.exe work and fetch your latest release?

Mpv themselves provide a test build for each pr,see https://github.com/mpv-player/mpv/actions/workflows/build.yml .
But I'm considering whether to release the product of each CI build(check mpv every hour for updated codes). This actually implies maintenance pressure. Releasing means that one should try to make each build successful, and when it fails one needs to find a way to fix it as soon as possible.

updater.ps1 works fine in my release and gets my updates. Just replace the original `installer/updater.ps1` with the updater.ps1 that came with my release.

**Reply by zhongfly** [reply]
@Infinitytreacher 
From now on, this repo will check the mpv repository every hour for updates.If there are any updates, it will start compiling and release when successful.
But I can only try my best to keep the build successful. If the build fails, you need to wait for shinchiro to update the compilation toolchain or until I have time to look for a solution
